.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - "6.5.0"
+env:
+  - CXX=g++-4.8
+addons:
+  apt:
+    sources:
+      - ubuntu-toolchain-r-test
+    packages:
+      - g++-4.8
+services:
+  - postgresql
+before_script:
+  - psql -c 'create database ce100_test;' -U postgres
```

Comment:
I know this is just for travis, but do we want to set a specific database that we want to test/configure all our plugins in?